### PR TITLE
feat: Support empty appid as interapp configure

### PR DESCRIPTION
**Pull request by 18202781743**
fallback to generic configuration if application is not changed,
and if generic configuration is changed, it will emit all related application's valueChanged.

Log: none
Influence: none
Change-Id: I90ad3bf631f69eb93445212a5cb6fab1c4ad1ec7

### CODE REVIEW

**Review by asterwyx — COMMENTED** [review]
+1

**Review by FeiWang1119 — APPROVED** [review]

File: debian/control [hunk 1]
```diff
@@ -8,7 +8,7 @@ Build-Depends:
  cmake,
  qtbase5-dev,
  libdtkcommon-dev,
- libdtkcore-dev,
+ libdtkcore-dev(>5.6.5),
```
Comment (asterwyx) [inline debian/control:11]: 这里是否是必要的。

**Review by asterwyx — CHANGES_REQUESTED** [review]

File: dconfig-center/dde-dconfig-daemon/dconfig_global.h [hunk 2]
```diff
@@ -25,17 +26,23 @@ using ConnRefCount = int;
...
+    return appid == NoAppId ? VirtualInterAppId : appid;
 }
 inline QString innerAppidToOuter(const QString &appid)
 {
-    return appid;
+    return appid == VirtualInterAppId ? NoAppId : appid;
+}
+inline bool isGenericResourceConn(const ConnKey &connKey)
+{
+    return connKey.startsWith("/" + VirtualInterAppId);
```
[the remark this answers is not in the document]
Comment (18202781743) [reply dconfig-center/dde-dconfig-daemon/dconfig_global.h:45]: 会出现，对于这种问题，下次提交整体上做个判断，

File: dconfig-center/dde-dconfig/main.cpp [hunk 3]
```diff
@@ -56,7 +56,8 @@ int main(int argc, char *argv[])
     QCommandLineOption listOption("list", QCoreApplication::translate("main", "list configure information with appid, resource or subpath."));
     parser.addOption(listOption);
 
-    QCommandLineOption appidOption("a", QCoreApplication::translate("main", "appid for the configure owner application."), "appid", QString());
+    QCommandLineOption appidOption("a", QCoreApplication::translate("main", "appid for a specific application.\n"
```
[the remark this answers is not in the document]
Comment (18202781743) [reply dconfig-center/dde-dconfig/main.cpp:59]: 在这里是一个选项，可以用小写

**Review by asterwyx — APPROVED** [review]

**Pull request merged**